test/gmonoid.lean

```diff
@@ -0,0 +1,30 @@
+/-
+Copyright (c) 2021 Eric Wieser. All rights reserved.
```

Comment:
```suggestion
Copyright (c) 2023 Eric Wieser. All rights reserved.
```
or has this waited more than a year? 😨

Reply:
This was written a long time ago in https://github.com/leanprover-community/mathlib/commit/5f8b45ef65ebe43c089fa9e473a62b6174136c68